provision.sh

```diff
@@ -14,31 +14,47 @@ echo "---------------------------------------------"
 echo "Running vagrant provisioning"
 echo "---------------------------------------------"
 
+# install heroku toolbelt
+echo "-------------- Installing heroku toolbelt -------------------------"
+#wget -O- https://toolbelt.heroku.com/install-ubuntu.sh | sh
+# These shell script snippets are directly taken from heroku installation script
+# We want to avoid the apt-get update
+# add heroku repository to apt
+echo "deb http://toolbelt.heroku.com/ubuntu ./" > /etc/apt/sources.list.d/heroku.list
```

Comment:
I usually run `apt-get update` after updating the `sources.list` and before installing something else. This something else might or might not depend on my recent change to the `sources.list`.

I don't know if that doe or does not, and that decision is better left to package manager. So, I run it them to be sure what I install next won't cause any problem with what I already have.